app/templates/components/header.html

```diff
@@ -1,5 +1,14 @@
-<nav class="navbar alt-background navbar-expand-lg navbar-expand-xl navbar-dark">
-  <a class="navbar-brand" href="/"><img src="/static/images/NA_Logo.png" class="img-fluid"></a>
+<!-- White navbar with NA logo -->
+<nav class="navbar navbar-collapse navbar-expand-lg navbar-expand-xl navbar-dark">
+  <div class="container-fluid">
+      <a class="navbar-brand" href="/">
+        <img alt="New Acropolis UK" src="/static/images/NA_Logo.png">
+      </a>
+      <div id="align-items-center h-100">United Kingdom</div>
```

Comment:
I'm not sure that changing from `row` to `id` is doing anything. 
I think if you strip out the `id` name and its values it will still behave in the same way